services/blockchain-indexer/shared/indexer/transactionProcessor/auth/registerMultisignature.js

```diff
@@ -22,17 +22,17 @@ const config = require('../../../../config');
 const {
 	getLisk32AddressFromPublicKey,
```

Comment:
Methods that are not associated with the DB and used by both indexer and dataService layers can be left wherever they used to be (in the shared directory).